src/camera.cpp

```diff
@@ -453,7 +453,7 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime, f32 tool_r
 
 	// Get FOV
 	f32 fov_degrees;
-	if (player->getPlayerControl().zoom && m_client->checkLocalPrivilege("zoom")) {
+	if (player->getPlayerControl().zoom && player->getCanZoom()) {
```

Comment:
getCanZoom -> canZoom

Reply:
no, getCanZoom is better as it's a noun

Reply:
don't you think textual reading is better ? if player->canZoom() ?

Reply:
Maybe `isZoomAllowed()`, `setZoomAllowed()`

Reply:
@juhdanad it sounds better, i agree :)

Reply:
'get' is consistent with all other 'getters', it would not be a 'getter' otherwise.
'getCanZoom' is shorter.